src/main/java/org/gridsuite/loadflow/server/LoadFlowService.java

```diff
@@ -45,37 +74,56 @@ private Network getNetwork(UUID networkUuid) {
     }
 
     LoadFlowResult loadFlow(UUID networkUuid, List<UUID> otherNetworksUuid, LoadFlowParameters parameters,
-                            String provider) {
+                            String provider, Optional<UUID> reportId, Optional<String> reportName) {
         LoadFlowParameters params = parameters != null ? parameters : new LoadFlowParameters();
         LoadFlowResult result;
 
         LoadFlow.Runner runner = LoadFlow.find(provider != null ? provider : DEFAULT_PROVIDER);
+        Reporter reporter;
         if (otherNetworksUuid.isEmpty()) {
             Network network = getNetwork(networkUuid);
```

Comment:
Why it is not  an option the report like merging view ?

Reply:
done